external/rules/pysm.sh

```diff
@@ -1 +1 @@
-pip install https://github.com/bthorne93/PySM_public/archive/2.1.0.tar.gz --prefix=@AUX_PREFIX@
```

Comment:
Note that I plan to delete the "external" directory in favor of using the cmbenv package after I move the dockerfile over there.  I've modified the pysm file in that repo to clone git master:  https://github.com/hpc4cmb/cmbenv/blob/master/pkgs/pysm.sh#L10